dot/services_test.go

```diff
@@ -221,7 +221,18 @@ func TestCreateRPCService(t *testing.T) {
 	sysSrvc, err := createSystemService(&cfg.System, stateSrvc)
 	require.NoError(t, err)
 
-	rpcSrvc, err := createRPCService(cfg, ns, stateSrvc, coreSrvc, networkSrvc, nil, sysSrvc, nil)
+	paramsRPC := createRPCServiceParams{
+		cfg,
+		ns,
+		stateSrvc,
+		coreSrvc,
+		networkSrvc,
+		nil,
+		sysSrvc,
+		nil,
+		nil,
+	}
```

Comment:
Use named fields please, thanks! Same below

Reply:
Done in 3de34da